client.go

```diff
@@ -193,7 +249,17 @@ func (c *Client) StreamWriteObject(ctx context.Context, body any) error {
 	if err != nil {
 		return err
 	}
-	return c.StreamWrite(ctx, tbl)
+	return c.streamWrite(ctx, types.INSERT, tbl)
+}
+
+// StreamDeleteObject is like [StreamDelete] to send the data into GreptimeDB, but schema is defined in the struct tag.
```

Comment:
In delete scenario, **send the data into GreptimeDB** should be updated to **delete the data from GreptimeDB**